model/Metadata/Service/ClassMetadataSearcher.php

```diff
@@ -36,14 +36,21 @@
 use oat\tao\model\Lists\Business\Service\GetClassMetadataValuesService;
 use oat\tao\model\search\ResultSet;
 use oat\tao\model\search\SearchProxy;
+use oat\tao\model\TaoOntology;
 use oat\taoAdvancedSearch\model\SearchEngine\Driver\Elasticsearch\ElasticSearch;
 use oat\taoAdvancedSearch\model\SearchEngine\Query;
 
 class ClassMetadataSearcher extends ConfigurableService implements ClassMetadataSearcherInterface
 {
+    use OntologyAwareTrait;
+
     private const BASE_LIST_ITEMS_URI = '/tao/PropertyValues/get?propertyUri=%s';
 
-    use OntologyAwareTrait;
+    private const UNACCEPTABLE_PROPERTIES = [
```

Comment:
@shpran we probably will need to play with this based on FF, so we will hide uniqueId and other translations properties if the feature flags is not ON:

- FEATURE_FLAG_TRANSLATION_ENABLED
- FEATURE_FLAG_UNIQUE_NUMERIC_QTI_IDENTIFIER